isce2_topsapp/__main__.py

```diff
@@ -95,6 +97,7 @@ def gunw_slc():
                         help='xmin ymin xmax ymax in epgs:4326', required=False)
     parser.add_argument('--estimate-ionosphere-delay', type=bool, default=False)
     parser.add_argument('--frame-id', type=int, default=-1)
+    parser.add_argument('--compute-solid-earth-tide', type=bool, default=False)
```

Comment:
The `type=bool` here won't work as expected; see https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/pull/95